subcommittee/coding-guidelines/mission-statement.md

```diff
@@ -0,0 +1,38 @@
+# Mission Statement - Coding Guidelines Subcommittee 
+
+Develop a community-vetted coding guideline for Rust in safety-critical applications. A living document that is updated as features are added to Rust and we learn more by doing.
+
+The following are relevant safety-critical coding guidelines. We are also interested in adopting knowledge obtained that may currently be in company-specific documents and incorporating these into the coding guideline.
```

Comment:
I was sort of thinking about this -- do we want to start with generalized coding guidelines for Rust, and then specialize into safety-critical guidelines (as two separate artifacts)? Or do we want to start from the outset focused only on a set of safety-critical focused guidelines? On the one hand, focusing seems desirable, but I believe some folks believed that we needed both. Just opening a discussion to see if anyone has thoughts here.

Reply:
Yeah, I hear you. IMHO these kinds of things are reasonable topics to bring to our first meeting to try to scope the work.

For my part, from what I've seen in the community and the desire from some in automotive I'd be interested in picking `unsafe` usage as a reasonable starting point to ship to get some momentum behind the group.